src/web/src/pages/CoursePage.vue

```diff
@@ -128,6 +104,15 @@ export default {
       }
       return precoreqtext;
     },
+    courseObj() {
+      let course = {};
+      for (const c of this.$store.state.courseList) {
+        if (c.name === this.courseName) {
+          course = c;
```

Comment:
Probably good to break out of loop here, or can use `courses.find((course) => course.name === name)`

Reply:
idk why I never think to use some of these built-in functions, thanks!